practice.css

```diff
@@ -1,3 +1,36 @@
-.details:checked + .label{
-    text-decoration-line: line-through;
-  }
\ No newline at end of file
+.display {
+  border: 0.2rem black solid;
+  width: 40%;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.details:checked + .label {
+  text-decoration-line: line-through;
+}
+
+list_items {
```

Comment:
check class naming conventions

Reply:
Removed unnecessary and ineffective styles.